docs/spfx/viva/get-started/actions/geolocation/GeolocationTutorial.md

```diff
@@ -0,0 +1,262 @@
+            "chooseLocationOnMap": true
+        }
+    },
+    {
+        "title": "Get my location",
+        "type": "VivaAction.GetLocation"
+    },
+    {
+        "title": "Show custom location",
+        "type": "VivaAction.ShowLocation",
+        "parameters": {
+            "locationCoordinates": {
+                "latitude": 27.98884062493244,
+                "longitude": 86.9249751
+            }
+        }
+    }
+  ]
+}
+```
+
+With this, we are providing two text-blocks to show the `Latitude` and `Longitude` of the location we get via `VivaAction.GetLocation`. In addition to these, we described three geolocation actions.
+
+After adding these actions, your Quick View would look like:
+
+![Card appearance after introducing changes in the quick-view](../../../../../../docs/images/viva-extensibility/geolocation/geoloactionQuickView.png)
+
+### Set up the state for our Adaptive Card Extension
+So far we have created our Card View and Quick View. If you do a `gulp serve` at this point, then you will be able to perform the actions that were described above.<br/>
+But now, let us take it a notch higher. <br/>
+We now wish to show the coordinates of user's current location or user's chosen location on the Quick View when the respective actions are executed. <br/>
+For this, we will leverage the two text-blocks that we had introduced earlier in the `QuickViewTemplate.json`.
+
+In order to do this, we will first introduce new states. First locate and open the following file in your project:
+`./src/adaptiveCardExtensions/geoLocation/GeoLocationAdaptiveCardExtension.ts`
+
+Here, add the following `states` to the `IGeoLocationAdaptiveCardExtensionState` interface:
+```json
+  latitude: string;
+  longitude: string;
+```  
+Next, in the `onInit` function, change `this.state={}` to
+```typescript
+  this.state = {
+    latitude: 'TBD',
+    longitude: 'TBD'
+  };
+```
+We will now make similar changes in Quick View as well. <br/>
+Locate and open the following file in your project:
+`./src/adaptiveCardExtensions/geoLocation/quickView/QuickView.ts`
+
+Add the following properties to the `IQuickViewData` interface:
+```json
+  latitude: string;
+  longitude: string;
+```
+and then add the following two lines in the returned object of `data` getter:
+```json
+      latitude: "Latitude: " + this.state.latitude,
+      longitude: "Longitude: " + this.state.longitude
+```
+
+
+
+### Implement the onAction function
+So far we have created defined our geolocation actions and wired in our states. Now we can finally implement the `onAction` function, which gives the ability to the Third Party Developer to decide what they wish to do with the location coordinates that the user has shared with them. <br/>
+For this, open the QuickView.ts file (`./src/adaptiveCardExtensions/geoLocation/quickView/QuickView.ts`) and import the `IActionArguments` interface, as follows: <br/>
+`import IActionArguments from @microsoft/sp-adaptive-card-extension-base` <br/>
```

Comment:
I would do full code block instead of inline to have beautiful colors for the text.
```typescript
import IActionArguments from @microsoft/sp-adaptive-card-extension-base
```

Reply:
done